pkg/common-controller/snapshot_controller.go

```diff
@@ -21,9 +21,9 @@ import (
 	"strings"
 	"time"
 
-	crdv1 "github.com/kubernetes-csi/external-snapshotter/pkg/apis/volumesnapshot/v1beta1"
-	"github.com/kubernetes-csi/external-snapshotter/pkg/utils"
-	"k8s.io/api/core/v1"
+	crdv1 "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+	"github.com/kubernetes-csi/external-snapshotter/v2/pkg/utils"
```

Comment:
These two lines above change the package version used in this file. This change isn't part of the upstream commit, I don't think we want those, do we?

Reply:
No, we don't want this.

Reply:
I removed this change, and the tests are passing as expected now. This one should be good for review and cherry-picking once merged.